src/Components/Blazor/testassets/HostedInAspNet.Server/HostedInAspNet.Server.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <!-- This is so that we add the FrameworkReference to Microsoft.AspNetCore.App -->
     <UseLatestAspNetCoreReference>true</UseLatestAspNetCoreReference>
+    <DisableImplicitComponentsAnalyzers>true</DisableImplicitComponentsAnalyzers>
```

Comment:
Is this essential to make the build pass because of the short-term versioning issues? If so could we also have an issue tracking the need to remove this later?

Reply:
I think pranav is taking care of it.

Reply:
https://github.com/dotnet/aspnetcore/issues/18841